packages/core/src/initialize.ts

```diff
@@ -12,11 +12,16 @@ export function initializeAgent(config: Config): Agent {
   const transports = initializeTransports(config);
   const api = initializeAPI(config, transports, metas);
 
+  const pause = () => transports.pause();
+  const unpause = () => transports.unpause();
+
   const agent = initializeGlobalAgent({
     config,
     metas,
     transports,
```

Comment:
Should we converts `transports` to an object and move `pause`/`unpause` methods there? It would make sense to me since that's what we are pausing.

Reply:
It is an object with `pause` / `unpause` :thinking:  I just wanted to have `pause()` method at top level `agent` as well for discoverability